core/modules/syncer.js

```diff
@@ -212,17 +209,15 @@ Syncer.prototype.updateDirtyStatus = function() {
 /*
 Save an incoming tiddler in the store, and updates the associated tiddlerInfo
 */
-Syncer.prototype.storeTiddler = function(tiddlerFields,hasBeenLazyLoaded) {
+Syncer.prototype.storeTiddler = function(tiddlerFields) {
 	// Save the tiddler
 	var tiddler = new $tw.Tiddler(tiddlerFields);
 	this.wiki.addTiddler(tiddler);
 	// Save the tiddler revision and changeCount details
 	this.tiddlerInfo[tiddlerFields.title] = {
-		revision: tiddlerFields.revision,
+		revision: (self.syncadaptor && self.syncadaptor.getTiddlerRevision && self.syncadaptor.getTiddlerRevision(tiddlerFields.title)) || tiddlerFields.revision,
```

Comment:
This line is ugly and it is used several times. So can we create a function for this one too. similar to `getSyncedTiddlers`

Reply:
Good thinking